model/group.go

```diff
@@ -218,6 +218,10 @@ func (group *Group) IsValidName() *AppError {
 			return NewAppError("Group.IsValidName", "model.group.name.invalid_length.app_error", map[string]any{"GroupNameMaxLength": GroupNameMaxLength}, "", http.StatusBadRequest)
 		}
 
+		if *group.Name == UserNotifyAll || *group.Name == ChannelMentionsNotifyProp || *group.Name == UserNotifyHere {
```

Comment:
Does this need to be checked when group name is updated?

Reply:
This function is also called, by `IsValidForUpdate`, when the group name is updated, so it is checked.